vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -47,16 +49,17 @@ async def create_speech(
             voice.settings = self.elevenlabs.VoiceSettings(
                 stability=self.stability, similarity_boost=self.similarity_boost
             )
-        url = ELEVEN_LABS_BASE_URL + f"text-to-speech/{self.voice_id}"
+        url = ELEVEN_LABS_BASE_URL + f"text-to-speech/{self.voice_id}/stream"
```

Comment:
since we're streaming this entirely into memory anyways, it doesn't make sense to use the `/stream` endpoint